tests/acceptance/bootstrap/SpacesContext.php

```diff
@@ -4834,4 +4810,59 @@ public function userHasExpiredTheMembershipOfUserFromSpace(
 		$rows['shareWith'] = $memberUser;
 		$this->featureContext->setResponse($this->shareSpace($user, $spaceName, $rows));
 	}
+
+	/**
+	 * @param string $user
+	 * @param string $resource
+	 * @param string $spaceName
+	 *
+	 * @return ResponseInterface
+	 * @throws GuzzleException|JsonException
+	 */
+	public function expireResourceShare(string $user, string $resource, string $spaceName): ResponseInterface {
+		$rows['expireDate'] = $this->featureContext->formatExpiryDateTime('Y-m-d\\TH:i:sP');
```

Comment:
```suggestion
		$rows['expireDate'] = $this->featureContext->formatExpiryDateTime('Y-m-d\\TH:i:sP');
```
Can we run tests locally with this format? 
Do we even care about timezone `P represents the timezone offset (e.g., +00:00, -05:00).` 
Could you try this format `$date->format('Y-m-d H:i:s');`